internal/envoy/bootstrap.go

```diff
@@ -199,3 +312,28 @@ func intOrDefault(i, def int) int {
 	}
 	return i
 }
+
+func writeConfig(filename string, config proto.Message) {
```

Comment:
We should return the error from this, letting it bubble all the way up the stack to be handled in the main setup.

Reply:
Done!